plugins/org.eclipse.xtend.core/META-INF/MANIFEST.MF

```diff
@@ -65,8 +64,6 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.6.0",
  org.eclipse.xtext.smap,
  org.apache.ant;bundle-version="1.7.1";resolution:=optional;x-installation:=greedy,
  org.objectweb.asm;bundle-version="[5.0.1,6.0.0)";resolution:=optional,
- org.eclipse.xtext.xbase.junit;resolution:=optional,
- org.eclipse.xtext.generator;resolution:=optional,
- org.eclipse.emf.mwe2.launch;bundle-version="2.7.1";resolution:=optional
+ org.eclipse.xtext.xbase.junit;resolution:=optional

```

Comment:
Yes, I think one of the existing in-memory-compilers uses some xbase.junit tools :(